sdk/appservice/arm-appservice/src/models/appServiceCertificateOrdersMappers.ts

```diff
@@ -84,7 +94,6 @@ export {
   FileSystemHttpLogsConfig,
   FunctionEnvelope,
   FunctionSecrets,
-  GeoDistribution,
```

Comment:
This also can indicate there's breaking changes

Reply:
Okay, I think it may be different from python sdk. I will change it. Thank you